init/main.go

```diff
@@ -36,7 +36,7 @@ var (
 
 	initBinary = "/sbin/init" // path to init binary inside the user's chroot
 
-	luksMappings []luksMapping // list of LUKS devices that booster unlocked during boot process
+	luksMappings []*luksMapping // list of LUKS devices that booster unlocked during boot process
```

Comment:
I thought about this, but wasn't sure if it was a dangerous change or what else it might modify so that is why I went with the more sloppy solution. I like this much better though, thanks!